docs/tutorials/get-a-user-by-name.md

```diff
@@ -0,0 +1,115 @@
+    cd <your-github-workspace>/to-do-service/api
+    # Run the service and monitor its database file for updates
+    json-server -w to-do-db-source.json
+    ```
+    
+    * Replace `<your-github-workspace>` with the path to your To-Do service clone on your system.
+
+2. Open the Postman app on your desktop.
+3. Click the **New** button in Postman and select **HTTP** to create a new request.
+4. Ensure **GET** is selected from the dropdown menu next to the request URL.
+5. In the URL field, enter the base URL for the To-Do service followed by the endpoint `/users`, for example:
+<!-- {% raw %} -->
+```
+    {{base_url}}/users
+```
+<!-- {% endraw %}) -->
+   
+6. To search for a specific user, append a question mark (`?`) to `/users` in the URL, followed by the relevant query parameters, and then click **Send** to submit the `GET` request to the To-Do service.
+   **Note:** Names are case-sensitive.
+   * To search by a specific last name, add the `last_name` parameter to the URL followed by an equals sign (`=`) and the desired last name.
+  
+      **Example Request:**      
+<!-- {% raw %} -->
+```
+      {{base_url}}/users?last_name=Doe
+```
+<!-- {% endraw %}) -->
+      **Example Response:**
+    
+      ```json
+      [
+          {
+              "last_name": "Doe",
+              "first_name": "Jamie",
+              "email": "jamie.doe@example.com",
+              "id": "1"
+          }
+      ]
+      ```
+    
+   * To search by a specific first name, add the `first_name` parameter to the URL followed by an equals sign (`=`) and the desired first name.
+ 
+      **Example Request:**
+<!-- {% raw %} -->
+```
+      {{base_url}}/users?first_name=Jamie
+```
+<!-- {% endraw %}) -->
+      **Example Response:**
+    
+      ```json
+      [
+          {
+              "last_name": "Doe",
+              "first_name": "Jamie",
+              "email": "jamie.doe@example.com",
+              "id": "1"
+          },
+          {
+              "last_name": "Smith",
+              "first_name": "Jamie",
+              "email": "jamie.smith@example.com",
+              "id": "2"
+          }
+      ]
+      ```
+    
+      **NOTE:**
+      In this example, two users are returned because two users with the first name Jamie exist.
+    
+   * To search by both first and last names, append `first_name=` followed by the first name, and `&last_name=` followed by the last name to the URL.
+ 
+      **Example Request:**
+<!-- {% raw %} -->
+```       
+      {{base_url}}/users?first_name=Jamie&last_name=Doe
+```
+<!-- {% endraw %}) -->    
+      **Example Response:**
+   
+      ```json
+      [
+          {
+              "last_name": "Doe",
+              "first_name": "Jamie",
+              "email": "jamie.doe@example.com",
+              "id": "1"
+          }
+      ]
+      ```
+
+**Note:** If no user matches the search criteria, the service will return an empty array (`[]`), indicating that there are no users with the specified names.
+
```

Comment:
```suggestion

## Next steps

```

To indicate the content of the section and that this is not part of the tutorial procedure.

A conclusion would be nice before this section to wrap up the tutorial, but if there's not much more to say at this point, then just a **Next steps** heading would accomplish the same effect.